irohad/ametsuchi/impl/postgres_wsv_command.hpp

```diff
@@ -68,6 +68,15 @@ namespace iroha {
           const std::string &permission_id) override;
 
      private:
+      template <typename T>
+      friend std::string try_quote(PostgresWsvCommand *,
+                                   std::initializer_list<T>) noexcept;
```

Comment:
also pass initializer list by ref or const ref

Reply:
Are you sure that you're understand what you say?

Reply:
ah, I missed the point. Okay, sorry.